### PR TITLE
TASK-44444: change UI ProductDetail

### PR DESCRIPTION
**Pull request by GouadriaHanen**

### CODE REVIEW

File: perk-store-webapps/src/main/webapp/vue-app/components/perk-store/ProductDetail.vue [hunk 1]
```diff
@@ -112,38 +144,9 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
...
-            grow
-            class="no-wrap mx-0">
-            <v-flex class="text-truncate title productCardTitle">
-              <span :title="product.title">{{ product.title }}</span>
-            </v-flex>
-            <v-flex class="primary--text headline text-right productCardPrice">
-              {{ product.price }} {{ symbol }}
-            </v-flex>
-          </v-layout>
+          <span :title="product.title">{{ product.title }}</span>
```
Comment (Rdidmohamedelhadi) [inline perk-store-webapps/src/main/webapp/vue-app/components/perk-store/ProductDetail.vue:147]: Use nowrap and ellipsis in the title

File: perk-store-webapps/src/main/webapp/vue-app/components/perk-store/ProductDetail.vue [hunk 2]
```diff
@@ -90,9 +101,29 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
...
+              class="productCardPending py-0"
+              @click="openProductDetail">
+              <v-hover v-if="userData.notProcessedOrders">
+                <v-chip
+                  slot-scope="{ hover: hoverPending }"
+                  :class="`${hoverPending && 'elevation-3'} userPendingOrders clickable`"
+                  @click="$emit('orders-list', product, null, true)">
+                  <v-icon
+                    :left="!$vuetify.rtl"
+                    color="#ffb441"
```
Comment (Rdidmohamedelhadi) [inline perk-store-webapps/src/main/webapp/vue-app/components/perk-store/ProductDetail.vue]: use the variable of this color

**Review by Rdidmohamedelhadi — CHANGES_REQUESTED** [review]

File: perk-store-webapps/src/main/webapp/vue-app/components/perk-store/ProductDetail.vue (hunk 1 above)
Comment (GouadriaHanen) [reply perk-store-webapps/src/main/webapp/vue-app/components/perk-store/ProductDetail.vue:147]: done !

File: perk-store-webapps/src/main/webapp/vue-app/components/perk-store/ProductDetail.vue (hunk 2 above)
Comment (GouadriaHanen) [reply perk-store-webapps/src/main/webapp/vue-app/components/perk-store/ProductDetail.vue]: done !

**Review by Rdidmohamedelhadi — APPROVED** [review]
LGTM

**Pull request merged**